stats_test.go

```diff
@@ -32,6 +32,8 @@ import (
 	monitoredrespb "google.golang.org/genproto/googleapis/api/monitoredres"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
 	"google.golang.org/grpc"
+	"google.golang.org/genproto/googleapis/api/distribution"
+	"fmt"
```

Comment:
run goimports to fix the import order (standard library should be at the top)

Reply:
Okay got it.